Boris_Kydrevich/.gitkeep

```diff
@@ -0,0 +1 @@
+
```

Comment:
Please exclude this from PR

Reply:
Sorry, i don't understand. This line or .gitkeep?

Reply:
> Please exclude this from PR

Sorry, i don't understand. This line or .gitkeep?

Reply:
File